frontend/class-twitter.php

```diff
@@ -161,6 +161,9 @@ protected function description() {
 		elseif ( WPSEO_Frontend::get_instance()->is_posts_page() ) {
```

Comment:
Good one for another moment, because refactoring this into a `switch` statement is pretty complex. I want to know what @omarreiss thinks about this.